screens/auth/LogInScreen.js

```diff
@@ -92,6 +94,13 @@ export default class LogInScreen extends React.Component {
         await updateCustomerPushTokens(customer, token);
         // Log in
         await this._asyncLogIn(customer.id);
+        // register this user in the Sentry logger
+        Sentry.configureScope(scope => {
```

Comment:
Need to move this before line 95. Right now it doesn't seem to be logging users. Same in Sign Up

Reply:
Also even with it moved, it doesn't seem to be picking up the phone numbers?

Reply:
I verified that this does register a user and associates any future sentry errors (although I had to hardcode it to verify) but it seems that `getAllCustomers` kept failing with the test accounts (both on prod + test), which is why it currently was not registering any information.
```
Sentry.configureScope(scope => {
         scope.setUser({
          id: '55',
           phoneNumber: formattedPhoneNumber,
       });
      }); 
```

Reply:
```
 Sentry.configureScope(scope => {
          scope.setUser({
           id: `${customerId}`,
            phoneNumber: formattedPhoneNumber,
        });
       }); 
```

@wangannie  @JustinMi -- let's move this as an else case to where it is guaranteed that it is a succcesful log-in since we will only need to set the user once: https://github.com/calblueprint/dccentralkitchen/blob/8c713862a78b5dbf3ceef6d5b951c5c2fa8113b1/screens/auth/LogInScreen.js#L124

Because the customerId is not saved in state, you would need to make customerId + customer `var variables` so they can be accessed in this else case

```
if (error !== '') {
        logAuthErrorToSentry({
          screen: 'LogInScreen',
          action: 'handleSubmit',
          attemptedPhone: formattedPhoneNumber,
          attemptedPass: password,
          error,
        });
   } else {
     Sentry.configureScope(scope => {
          scope.setUser({
           id: `${customerId}`,
            phoneNumber: formattedPhoneNumber,
        });
       }); 
}
```